feature-libs/product-configurator/rulebased/components/price-async/configurator-price-async.component.html

```diff
@@ -0,0 +1,11 @@
+<ng-container *ngIf="configuration$ | async as configuration">
+  <ng-container *ngIf="getPriceDetails(configuration) as priceDetails">
+    <div
+      *ngIf="displayPrice(priceDetails)"
+      [ngClass]="styleClass"
+      [attr.aria-label]="'configurator.a11y.valueSurcharge' | cxTranslate"
+    >
+      {{ getDisplayPrice(priceDetails) }}
+    </div>
+  </ng-container>
+</ng-container>
```

Comment:
Only CPS uses ASYNC prices and CPS only supports value prices, but no price formulars. So this component is much simpler, than the sync pricing component which also needs to support price formulars for CPQ.